### PR TITLE
Be login signup 9

### PR DESCRIPTION
**Pull request by timegambit**
Will need to add a .env variable of the form SECRET_KEY=(some random string) for singing JWT tokens.
You can run these commands in terminal for a random string:
```
node
require("crypto").randomBytes(64).toString('hex')
```

### CODE REVIEW

File: server/routes/register.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          );
+          res.cookie("token", token, { httpOnly: true });
+          res.status(201).send();
+        }
+      }
+    }
+});
+
+function isValidPassword(pw) {
+  return pw.length > 6;
```
Comment (aeciorc) [inline server/routes/register.js]: \>=

Comment (aeciorc) [reply server/routes/register.js]: Let's validate the email as well

**Comment by timegambit** [comment]
I added a validate-body middleware shared between the login and register routes. It uses the express-validator package to validate and sanitize data (trim, normalize). Others may choose to implement their own validation in this file (it may be useful for validating fields in the profile model @ethanrosenberg if its needed).

**Review by aeciorc — APPROVED** [review]
Looks good! Nice job with the error handling

**Pull request merged**